src/Sulu/Component/DocumentManager/Event/CopyLocaleEvent.php

```diff
@@ -30,12 +25,10 @@ class CopyLocaleEvent extends AbstractMappingEvent
      * @param string $locale
      * @param string $destLocale
      */
-    public function __construct($document, $locale, $destLocale)
+    public function __construct($document, $locale, private $destLocale)
     {
         $this->document = $document;
         $this->locale = $locale;
```

Comment:
we may should have a look at this `AbstractMappingEvent` in a seperate pull request. To retype all through and tell what is nullable and what could be part of `AbstractMappingEvent::__construct`.